webpack.config.js

```diff
@@ -212,10 +212,7 @@ const webpackConfig = {
 			SassConfig.loader( {
 				preserveCssCustomProperties: true,
 				includePaths: [ path.join( __dirname, 'client' ) ],
-				prelude: `@import '${ path.join(
-					__dirname,
-					'client/assets/stylesheets/shared/_utils.scss'
-				) }';`,
+				prelude: `@import '~@automattic/calypso-sass-prelude/sass/utils';`,
```

Comment:
Should we also `packages/calypso-ui` accordingly? (See https://github.com/Automattic/jetpack/pull/13490#issuecomment-540520866)